Manifests/ios.json

```diff
@@ -31,6 +31,12 @@
       "label": "Send User Data",
       "default": 1,
       "tooltip_text": "Sends any user data collected in the app. Per Google&apos;s policies, we will not send any user data identified as PII (Personally Identifiable Information)."
+    },
+    {
+      "type": "text",
+      "key": "user_id",
```

Comment:
Should be app family id also in the type, label and tooltip

Reply:
No. This was secified by product. We want it to be difficult for a Zapper to configure, as to configure has legal implies. So we don't want to document anything or change change the key/tool tip.